python/tvm/relay/op/transform.py

```diff
@@ -1450,6 +1450,75 @@ def sparse_reshape(sparse_indices, prev_shape, new_shape):
     return TupleWrapper(_make.sparse_reshape(sparse_indices, prev_shape, new_shape), 2)
 
 
+def segment_sum(data, segment_ids, num_segments=None):
+    """
+    Computes the sum along segment_ids along axis 0. If multiple segment_ids reference the same
+    location their contributions add up.
+    result[index] = Σi... data[i...] where index = segment_ids[i]
```

Comment:
This expression seems wrong in the multidimensional case. Shouldn't it be `result[index, j, k, ...] = \sum =_i data[i, j, k,..]`

Reply:
Yes, corrected it now.